### PR TITLE
[ci skip] documentation for #36589

### PR DESCRIPTION
**Pull request by yskkin**
### Motivation / Background

This is missing documentation for #36589.

### Additional information

I do not know rationale behind 58a0c2ea98cd0d46965e4ce7346a1159dce0dbd8, but seems to be incorrect.

### Checklist

Before submitting the PR make sure the following are checked:

* [ ] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [ ] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [ ] Tests are added or updated if you fix a bug or add a feature.
* [ ] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -632,8 +633,8 @@ actions automatically. Below are some of the actions that `change` supports:
 * [`rename_index`][]
 * [`rename_table`][]
 
-[`change_table`][] is also reversible, as long as the block does not call `change`,
-`change_default` or `remove`.
+[`change_table`][] is also reversible, as long as the block does not call irreversible operation
+which does not correspond to the operations listed above.
```
Comment (yskkin) [inline guides/source/active_record_migrations.md:637]: `change_default` or `remove` may be reversible if proper option is given. Wording can be improved though.

Comment (jonathanhefner) [inline guides/source/active_record_migrations.md:637]: Would it be more clear without the double negative?

```suggestion
[`change_table`][] is also reversible, as long as the block only calls
reversible operations like the ones listed above.
```

**Review by jonathanhefner — COMMENTED** [review]
Just a few small tweaks. :smiley:

**Comment by jonathanhefner** [comment]
Thank you, @yskkin! :smile:

**Pull request merged**